js/main.js

```diff
@@ -35,12 +44,12 @@ function toList(collection) {
 function calculateTotalMats(elementAmount) {
     let collection = {};
     for (let key of Object.keys(crafting)) { // key = key of crafting Object
-        for (let value of crafting[key]) { // value = value associated with key from crafting Object
+        for (let value of crafting[key].materials) { // value = value associated with key from crafting Object

```

Comment:
The comment is outdated (This is also a place where your variable name could be more descriptive)

Reply:
Fixed naming, comment removed